src/build.d

```diff
@@ -1580,7 +1581,13 @@ class BuildRule
             }
             else if (command.length)
             {
-                command.run;
+                import std.ascii : newline;
+                const output = command.run.strip;
+                if (output.length > 0) {
+                    auto ruleName = name.length ? name : "log";
+                    // add [<name>] prefix to all log messages
+                    output.splitter(newline).map!(e => "[" ~ ruleName ~ "] " ~ e).joiner(newline).writeln;
```

Comment:
```suggestion
                    output.splitter(newline).each!(ln => writeln('[', ruleName, "] ", ln));
```

is simpler I guess

Reply:
This risks interleaving with other log messages from other processes (build.d might executes multiple dependencies at the same time).